frodo-android-sample/src/test/java/com/fernandocejas/example/frodo/sample/ObservableSampleTest.java

```diff
@@ -0,0 +1,20 @@
+package com.fernandocejas.example.frodo.sample;
```

Comment:
I do not think we need this test for the sample. The main reason is that the samples are likely to be changed and this is only testing what the observable emits which makes our code very fragile to change. 
By the way, tests means more code to maintain and in this case, what is important is the domain of the library. 

Can you remove this test and test dependencies above in ```build.gradle``` so I can merge your fix?